product_docs/docs/biganimal/release/using_cluster/05c_upgrading_log_rep.mdx

```diff
@@ -0,0 +1,225 @@
+(1 row)
+```
+
+Check that both instances are the expected versions.
+
+### Migrate the database schema 
+
+Next, confirm the database schema you want to migrate. While logged into the old instance, use the following SQL to look at the table data:
+
+```sql
+/dt+;
+```
+
+Here is a sample database schema for this example:
+
+```
+                                           List of relations
+ Schema |       Name       | Type  |   Owner   | Persistence | Access method |    Size    | Description 
+--------+------------------+-------+-----------+-------------+---------------+------------+-------------
+ public | pgbench_accounts | table | edb_admin | permanent   | heap          | 1572 MB    | 
+ public | pgbench_branches | table | edb_admin | permanent   | heap          | 8192 bytes | 
+ public | pgbench_history  | table | edb_admin | permanent   | heap          | 0 bytes    | 
+ public | pgbench_tellers  | table | edb_admin | permanent   | heap          | 120 kB     | 
+```
+
+
+Then, from a Postgres client, copy over the database schema from the old instance using the `pg_dump` command:
+
+```
+pg_dump --schema-only  -h <old_biganimal_host> -U <old_biganimal_username> -d <old_biganimal_databasename> | psql -h <new_biganimal_host> -U <new_biganimal_username> -d <new_biganimal_databasename>
+```
+
+The `pg_dump --schema-only` command exports the schema (structure) of the existing database without the data. It's then piped into `psql` to import this schema into the new Postgres instance. This prepares the target database with the necessary structure to hold the data.
+
+Finally, log into the new instance and confirm that the database schema migrated correctly. Run `\dt+;` in this example again and you see the database schema migrated to the new instance as expected:
+
+```
+                                         List of relations
+ Schema |       Name       | Type  |   Owner   | Persistence | Access method |  Size   | Description 
+--------+------------------+-------+-----------+-------------+---------------+---------+-------------
+ public | pgbench_accounts | table | edb_admin | permanent   | heap          | 0 bytes | 
+ public | pgbench_branches | table | edb_admin | permanent   | heap          | 0 bytes | 
+ public | pgbench_history  | table | edb_admin | permanent   | heap          | 0 bytes | 
+ public | pgbench_tellers  | table | edb_admin | permanent   | heap          | 0 bytes | 
+```
+
+While the schema looks the same on the new instance, the "Size" column shows 0 bytes of data, confirming that only the schema has been migrated so far.
+
+### Setting up Publication 
+
+To set up the publication, log into the old BigAnimal instance and create a publication:
+
+```sql
+CREATE PUBLICATION <pub_name>;
+```
+
+In this example:
+
+
+```sql
+CREATE PUBLICATION v12_pub;
+```
+
+The expected output is `CREATE PUBLICATION`.
+
+Next, configure the publication to include the specific tables you want replicated on the new instance:
```

Comment:
Add tables that you want to replicate to your target instance:

Reply:
Got it. 👍